h/migrations/versions/bd0cc0e6ed54_create_the_moderation_status_column.py

```diff
@@ -0,0 +1,28 @@
+"""Create the moderation_status column."""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "bd0cc0e6ed54"
+down_revision = "1d26b96db4af"
+
+
+def upgrade() -> None:
+    moderation_status_type = postgresql.ENUM(
```

Comment:
From the PoC:

This is an enum in postgres.

I generally try to avoid postgres enum because alembic is not 100% aware of them.

Here I reckon we should use them, they are very efficient storage-wise and the manual migrations we might have to write on them are actually trivial, is just that alembic doesn't pick them up:

```
ALTER TYPE moderationstatus ADD VALUE 'PENDING';   
```


Note that this a nullable field. This is necessary to create the new column but also the code assumes that null is either a private annotation or an approved one.



https://github.com/hypothesis/h/pull/9461#discussion_r2034698836